pycoal/mineral.py

```diff
@@ -142,10 +153,100 @@ def SAM(library, image_file_name, classified_file_name, scores_file_name=None, t
                 'map info': image.metadata.get('map info')
             })
 
+def avngDNN(image_file_name, classified_file_name, model_file_name, class_names=None, scores_file_name=None, in_memory=False):
+    """
+    This callback function takes a Keras model, trained to classify pixels from AVIRIS-NG
+    imagery, and assigns a class for every single pixel of the input image.
+
+    Parameter 'scores_file_name' optionally receives the path to where to save
+    an image that holds all the SAM scores yielded for each pixel of the 
+    classified image. No score image is create if not provided.
+
+    In order to improve performance on systems with sufficient memory,
+    enable the optional parameter to load entire images.
+
+    Args:
+        image_file_name (str):          filename of the image to be classified
+        classified_file_name (str):     filename of the classified image
+        model_file_name (str):          filename of the Keras model used to classify
```

Comment:
I thought we were going to use [Apache MXNet](https://mxnet.apache.org)? Regardless, we need to provide documentation on ***how*** a model is initially created.

Reply:
Due to delays related to my unsuccessful trials to get other example scenes, there was not enough time to make also the MXNet models, but I'm going to make another callback function to work with them, too.

Reply:
@lewismc , I've finished the jupyter notebook tutorial cited in [my last week report](https://github.com/ESIPFed/GSoC/wiki/ISSUE-10-Reporting#week-13-6th-august--14th-august):

[https://www.lapig.iesa.ufg.br/drive/index.php/s/JbnPcl0GDHezPO0](https://www.lapig.iesa.ufg.br/drive/index.php/s/JbnPcl0GDHezPO0)

 This is part of the documentation but I don't know where to pla

Reply:
Please just create a new repository and name it pycoal_examples, you can add it there. Thanks

Reply:
Here's the repo, @lewismc : https://github.com/capstone-coal/pycoal_examples/blob/master/tutorial.ipynb

Reply:
Great.